detox/e2e/test/channels/join_from_permalink.e2e.js

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
I've created a new test case in TM4J as [MM-T3471](https://mattermost.atlassian.net/projects/MM?selectedItem=com.atlassian.plugins.atlassian-connect-plugin%3Acom.kanoah.test-manager__main-project-page#!/testCase/MM-T3471) in `/Messaging` test folder (same as with related [MM-T2224](https://mattermost.atlassian.net/projects/MM?selectedItem=com.atlassian.plugins.atlassian-connect-plugin%3Acom.kanoah.test-manager__main-project-page#!/testCase/MM-T2224)). With that, please move this file to `detox/e2e/test/messaging`.

Also, I thought `permalink` refers to link to a post/message. For channel, I believe it's simply called channel URL or link. If you agree, could you please update the filename?

Reply:
I think they're all permalinks just different types: channel, post, etc but I may be wrong. So I think it should be prefixed with `channel_` to differentiate